packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -164,7 +163,6 @@ export const useProductFullDetail = props => {
         productType
     );
 
-    const [, { toggleDrawer }] = useAppContext();
```

Comment:
Did we intentionally remove this functionality in the scope of the epic? If you're on desktop and you add a product to the cart should the new mini cart open? If you're on mobile and you add a product to the cart should you go straight to cart?

Reply:
> If you're on desktop and you add a product to the cart should the new mini cart open?
> If you're on mobile and you add a product to the cart should you go straight to cart?

No in both cases. You have to click the cart trigger to open the mini cart (desktop) or go to the cart page (mobile).

Reply:
OK - I'm good with this, even if we do end up feature flagging the new mini cart.